src/tests/sqlsmith/src/runner.rs

```diff
@@ -188,6 +189,43 @@ pub async fn run(client: &Client, testdata: &str, count: usize, seed: Option<u64
     tracing::info!("[EXECUTION SUCCESS]");
 }
 
+/// Differential testing for batch and stream
+pub async fn run_differential_testing(
+    client: &Client,
+    testdata: &str,
+    count: usize,
+    seed: Option<u64>,
+) -> Result<()> {
+    let mut rng = generate_rng(seed);
+
+    set_variable(client, "RW_IMPLICIT_FLUSH", "TRUE").await;
```

Comment:
Maybe using an explicit flush can be more efficient.

Reply:
Sorry I couldn't quite get it, why would calling flush per insert be more efficient?

---

I think it is worth to trade-off efficiency for simplicity.
I don't think inserts take very long to run. Prefer to set this variable and forget about calling `flush` each time. Will be less error prone.

Reply:
Implicit flush will make each `INSERT` be blocked by barrier interval, so it might be very slow if there are many `INSERT`s. 👀  Correct me if I'm wrong.

Reply:
> only run 1 INSERT per table, but batch many rows into that INSERT statement.

makes sense